components/layout.js

```diff
@@ -6,20 +6,34 @@ import {
   ChatAlt2Icon,
   UserCircleIcon,
   XIcon,
+  LogoutIcon,
+  BriefcaseIcon
 } from "@heroicons/react/outline";
 import clsx from "clsx";
 import { SidebarLink, Sidebar, OpenSidebar } from "./sidebar";
 import { Profile } from "./profile";
-import { LogoutButton } from "./auth";
 
 // TODO: TD-082321: Setup real navigation with links
 const navigation = [
   { name: "Home", href: "/feed", icon: HomeIcon, current: true },
-  { name: "About", href: "/about", icon: QuestionMarkCircleIcon, current: false },
+  {
+    name: "About",
+    href: "/about",
+    icon: QuestionMarkCircleIcon,
+    current: false,
+  },
   { name: "Profile", href: "/profile", icon: UserCircleIcon, current: false },
   { name: "Messages", href: "/messages", icon: ChatAlt2Icon, current: false },
+  { name: "Legal", href: "/legal", icon: BriefcaseIcon, current: false },
```

Comment:
I don't think this should belong on the nav.

Reply:
On the bottom then? (i.e. a footer?)

Reply:
Yes a footer would be better.